src/fr/articles/landmarks-aria/index.njk

```diff
@@ -1,137 +1,235 @@
+          <td><code>role="search"</code></td>
+        </tr>
+        <tr>
+          <td><code>main</code></td>
+          <td><code>role="main"</code></td>
+        </tr>
+          <tr>
+          <td><code>form</code></td>
+          <td><code>role="form"</code></td>
+        </tr>
+        <tr>
+          <td><code>section</code></td>
+          <td><code>role="region"</code></td>
+        </tr>
+        <tr>
+          <td><code>aside</code></td>
+          <td><code>role="complementary"</code></td>
+        </tr>
+        <tr>
+          <td><code>footer</code></td>
+          <td><code>role="contentinfo"</code> seulement dans certains cas (voir <a href="#usages">Usages</a> ci-dessous)</td>
+        </tr>
+      </tbody>
+  </table>
+
+<h3 id="usages" class="mt-4">Usages</h3>
+  <h4 id="header">header</h4>
+  <p>La balise <code>header</code> (ou l'attribut de région <abbr>ARIA</abbr> <code>role="header"</code>) permet d'identifier l’en-tête de la page. Cela est vrai uniquement si cet élément est placé dans une balise <code>body</code>,
+      soit comme enfant direct, soit dans une balise <abbr>HTML</abbr> autre que <code>article</code>, <code>aside</code>, <code>main</code>, <code>nav</code> et <code>section</code>.</p>
+  <p>De manière générale et pour simplifier, il est conseillé d'utiliser la balise <code>header</code> (ou l'attribut de région <abbr>ARIA</abbr> <code>role="header"</code>) une seule fois par page.</p>
+
+  <h4 id="nav">nav</h4>
+  <p>La balise <code>nav</code> (ou l'attribut attribut de région <abbr>ARIA</abbr> <code>role="navigation"</code>) permet d’identifier les principaux éléments de navigation internes au site. 
+  <p>Il est conseillé d'utiliser au maximum 3 balises <code>nav</code> (ou <code>role="navigation"</code>) dans une même page.</p>
+  <p>Lorsque plusieurs <code>nav</code> sont utilisées, chacune doit avoir un nom accessible unique qui permettra de les identifier (voir <a href="#bonnes_pratiques">Bonnes pratiques</a>).</p>
+ 
+  <h4 id="search">role="search"</h4>
+  <p>L'attribut de région <abbr>ARIA</abbr> <code>role="search"</code> (pas de balise <abbr>HTML</abbr> équivalente) est à utiliser pour identifier un formulaire de recherche dans le site ou pour filtrer du contenu présent dans la page.</p>
+ 
+  <h4 id="main">main</h4>
+  <p>La balise <code>main</code> (ou l'attribut de région <abbr>ARIA</abbr> <code>role="main"</code>) permet d'identifier le contenu principal de la page.</p>
+  <p>Cet élément, dont la présence est obligatoire dans chaque page <abbr>HTML</abbr>, n'est à utiliser qu'une seule fois par page et uniquement comme enfant des balises <abbr>HTML</abbr> <code>body</code>, <code>html</code>, <code>div</code> et <code>form</code>. </p>
+  <p>Note : si une page <abbr>HTML</abbr> contient un <code>iframe</code>, celui-ci peut également contenir une balise <code>main</code> (ou un attribut <code>role="main"</code>) car c'est une page autonome.</li></p>
```

Comment:
```suggestion
  <p>Note : si une page <abbr>HTML</abbr> contient un <code>iframe</code>, celui-ci peut également contenir une balise <code>main</code> (ou un attribut <code>role="main"</code>) car c'est une page autonome.</p>
```

Reply:
Not sure why there is a `</li>` here? Couly check it please?

Reply:
You're right, inappropriate autocompletion. Corrected.